app/assets/stylesheets/layout.scss

```diff
@@ -1953,28 +1935,6 @@ table {
     width: 100%;
   }
 
-  .debates-inner,
-  .proposals-inner,
-  .budget-investments-inner {
-    max-height: rem-calc(350);
-
-    @include breakpoint(small) {
-      max-height: rem-calc(400);
-    }
-
-    h4 {
-      font-size: rem-calc(18);
-      margin-bottom: $line-height;
-      margin-top: 0;
-      min-height: 0;
-    }
-
-    h5 {
-      font-size: $small-font-size;
-      text-align: left;
-    }
-  }
-
```

Comment:
This code might still be used in the `recommended_carousel` partial, **with the exception** of the `budget-investments-inner` class, which isn't used since commit c6a9ffc19c75.

```erb
<div class="small-12 medium-6 column <%= key %> ">
  <div class="card small-centered <%= key %>-inner">

    <h3><%= t("welcome.recommended.#{key.underscore}.title") %></h3>
```